slack/src/commonMain/kotlin/work/socialhub/planetlink/slack/action/SlackActionHelper.kt

```
@@ -226,27 +228,70 @@ internal class SlackActionHelper(
-            val channelUser = channel.user
-            val channelId = channel.id
-            if (channelUser != null && channelId != null) {
-                memberMap[channelId] = listOf(userMe.id!!.value<String>(), channelUser)
+        val userMeId = userMe.id!!.value<String>()
+
+        for (channel in response.channels.orEmpty()) {
+            if (!SlackMapper.isVisibleThread(channel, userMeId)) continue
+            val channelId = channel.id ?: continue
+            try {
+                memberMap[channelId] = getThreadMemberIds(channel, userMeId)
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)